Source/options.cpp

```diff
@@ -581,24 +581,38 @@ string_view OptionCategoryBase::GetDescription() const
 DiabloOptions::DiabloOptions()
     : OptionCategoryBase("Diablo", N_("Diablo"), N_("Diablo specific Settings"))
     , intro("Intro", OptionEntryFlags::OnlyDiablo, N_("Intro"), N_("Enable/disable Intro cinematic."), true)
+    , splash("Splash", OptionEntryFlags::OnlyDiablo, N_("Splash"), N_("Shown splash screen at startup."), StartUpSplash::LogoAndTitleDialog,
+          {
+              { StartUpSplash::LogoAndTitleDialog, N_("Logo and Title Dialog") },
+              { StartUpSplash::TitleDialog, N_("Title Dialog") },
+              { StartUpSplash::None, N_("None") },
+          })
 {
 }
 std::vector<OptionEntryBase *> DiabloOptions::GetEntries()
 {
 	return {
 		&intro,
+		&splash,
 	};
 }
 
 HellfireOptions::HellfireOptions()
     : OptionCategoryBase("Hellfire", N_("Hellfire"), N_("Hellfire specific Settings"))
     , intro("Intro", OptionEntryFlags::OnlyHellfire, N_("Intro"), N_("Enable/disable Intro cinematic."), true)
+    , splash("Splash", OptionEntryFlags::OnlyHellfire, N_("Splash"), N_("Shown splash screen at startup."), StartUpSplash::LogoAndTitleDialog,
+          {
+              { StartUpSplash::LogoAndTitleDialog, N_("Logo and Title Dialog") },
+              { StartUpSplash::TitleDialog, N_("Title Dialog") },
+              { StartUpSplash::None, N_("None") },
+          })

```

Comment:
I don't feel there is a need for this per game option, the one in `sgOptions.Diablo.splash` will suffice.

Reply:
This mimic what we currently have for the `intro` option.

I have no problem with changing it, only two questions:
- Should we let it stay in `Diablo` section or move it somewhere else? Cause I thought that the `Diablo` and `Hellfire` settings are specific for the corresponding game.
- Should we also move/combine the `intro` setting for diablo and hellfire?

Reply:
its fine to have it in the Diablo section, else we would have to create a new one (which is also fine)

intos are different an automatically change setting, leave them seperate

Reply:
Currently intro is [not automatically](https://github.com/diasurgical/devilutionX/pull/3437#issuecomment-964590894) changed.
> Intro changed to on/off option (default on) and will be on until it's switched in the settings menu
Did I misinterpreted the comments ([Nr1](https://github.com/diasurgical/devilutionX/pull/3437#issuecomment-962715359) and [Nr2](https://github.com/diasurgical/devilutionX/pull/3437#issuecomment-962760168)) wrong?

What do you think about a new category "Startup" (or another name) that combines intro, splash, (later) game mode (diablo or hellfire) and perhaps also fullversion and shareware? Then we wouldn't show the diablo or hellfire category in the settings menu at all.

Reply:
I wasn't very specific and was more in general agreement, but not necessarily on all details of the comment. Sorry for not giving more details :)

Some thing like this could work
0: off
1: play once
2: always on

But I doubt anyone would use 2, so that is optional imo

> What do you think about a new category "Startup" (or another name) that combines intro, splash, (later) game mode (diablo or hellfire) and perhaps also fullversion and shareware? Then we wouldn't show the diablo or hellfire category in the settings menu at all.

Sounds good.

Reply:
I moved intro and splash to a new `StartUp`-Category that don't make a difference between diablo and hellfire.
Intro has now a "Once"-Option (Isn't visible in UI-Settings. My thought was, that if a user want's to explicitly enable it, the user always wants to enable it and not only once; can change it if wanted).
Diablo and Hellfire category is not visible in UI-Settings.